apps/server/src/infra/authorization-client/authorization-api-client/api/authorization-api.ts

```diff
@@ -0,0 +1,159 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * HPI Schul-Cloud Server API
```

Comment:
Das sollten wir im den Swagger configs noch ändern. 
.setTitle('HPI Schul-Cloud Server API')  
.setDescription('This is v3 of HPI Schul-Cloud Server. Checkout /docs for v1.')

HPI Schul-Cloud ersetzten durch Schulcloud-Verbund-Software

Reply:
Fixed by renaming `HPI Schul-Cloud` to `Schulcloud-Verbund-Software` in `src/swagger.js`!